Baby Monitor/Extensions/UserDefaults/UserDefaults+SoundDetectionMode.swift

```diff
@@ -10,14 +10,31 @@ extension UserDefaults {
        return "SOUND_DETECTION_KEY"
     }
 
+    private static var noiseLevelKey: String {
+       return "NOISE_LEVEL_KEY"
+    }
+
     /// A current mode that is set in the app.
     static var soundDetectionMode: SoundDetectionMode {
         get {
             let rawValue = UserDefaults.standard.string(forKey: soundDetectionModeKey) ?? Constants.defaultSoundDetectionMode.rawValue
-            return SoundDetectionMode(rawValue: rawValue)!
+            return SoundDetectionMode(rawValue: rawValue) ?? Constants.defaultSoundDetectionMode
         }
         set {
             UserDefaults.standard.set(newValue.rawValue, forKey: soundDetectionModeKey)
         }
     }
+
+    static var noiseLoudnessFactorLimit: Int {
+        get {
+            if UserDefaults.standard.object(forKey: noiseLevelKey) == nil {
+                return Constants.loudnessFactorLimit
+            } else {
+                return UserDefaults.standard.integer(forKey: noiseLevelKey)
+            }
```

Comment:
I think you can remove `else`:
```
 if UserDefaults.standard.object(forKey: noiseLevelKey) == nil {
     return Constants.loudnessFactorLimit
 } 
 return UserDefaults.standard.integer(forKey: noiseLevelKey)
```
But it's up to you if its more readable like that 😄

Reply:
Actually, in this case, I prefer leaving it with else :)

Reply:
Actually, in this case, I prefer leaving it with else :)